daily_milestone.go

```diff
@@ -199,10 +217,11 @@ func createMilestones(baseURL string, token string, projectID string, milestones
 
 func main() {
 	// Declaring variables for flags
-	var token, baseURL, namespace, project string
+	var token, baseURL, namespace, project, milestoneTime string
 	var advance int
 	// Command Line Parsing Starts
 	flag.StringVar(&token, "Token", "jGWPwqQUuf37b", "Gitlab api key/token.")
+	flag.StringVar(&milestoneTime, "milestoneTime", "Daily", "to set milestone to daily, weekly or monthly.")
```

Comment:
The description of the flag differs a bit from the others. Why not just say. "Set milestone to daily, weekly or monthly"

Reply:
Also all flags are uppercase, when userfacing. "milestoneTime" is lowercase. Inconsistent for the user.